public/app/core/components/PageHeader/PageHeader.tsx

```diff
@@ -22,10 +22,16 @@ const SelectNav = ({ children, customCss }: { children: NavModelItem[]; customCs
 
   return (
     <div className={`gf-form-select-wrapper width-20 ${customCss}`}>
-      <div className="dropdown">
-        <div className="gf-form-input dropdown-toggle" data-toggle="dropdown">
+      <div role="menu" className="dropdown">
```

Comment:
Thanks, makes sense, I had added first the `option` role + `aria-selected` to the div in the hope that it would become focusable 😄 , I'll remove them and move the `menu` role to the `ul`, you're right using the arrows instead of tabbing will be more intuitive in the list 👍